nextcloudappstore/core/views.py

```diff
@@ -297,3 +300,60 @@ def get_context_data(self, **kwargs):
         context = super().get_context_data(**kwargs)
         context['form'] = AppRegisterForm()
         return context
+
+
+class AppEditView(LoginRequiredMixin, TemplateView):
```

Comment:
This view should not be shown to anyone else but the current owner of the app. This must be fixed if we choose to proceed with this approach.